.gitignore

```diff
@@ -5,7 +5,7 @@
 .lambda_hashes.json
 terraform.plan
 .terraform
-terraform/releases
+shared_infra/releases
```

Comment:
Minor: maybe we could rename this directory to `.releases`?

Then it’s hidden from view, and we get the same effect if we use this pattern in other stacks.